lib/fs/filesystem.go

```diff
@@ -22,23 +22,38 @@ type Filesystem interface {
 	DirNames(name string) ([]string, error)
 	Lstat(name string) (FileInfo, error)
 	Mkdir(name string, perm FileMode) error
+	MkdirAll(name string, perm FileMode) error
 	Open(name string) (File, error)
+	OpenFile(name string, flags int, mode FileMode) (File, error)
 	ReadSymlink(name string) (string, error)
 	Remove(name string) error
+	RemoveAll(name string) error
 	Rename(oldname, newname string) error
 	Stat(name string) (FileInfo, error)
 	SymlinksSupported() bool
 	Walk(root string, walkFn WalkFunc) error
+	Show(name string) error
```

Comment:
The name `Show()` makes perfect sense in context, but is somewhat unclear out of context - does it pop up a GUI preview of the file maybe? I wonder if `Unhide()` would be better, even if it's a double negative?